### PR TITLE
[#960]Refactor Authorship Css

### PR DESCRIPTION
**Pull request by Tejas2805**
Fixes part of #960 

```
The JS/HTML code of each component has already rested in its 
own file, but all the CSS are in one file. 

Let's refactor the Authorship CSS into a new file for better 
scalability and readability.
```

### CODE REVIEW

**Comment by Tejas2805** [comment]
@fzdy1914 If you are okay with the refactoring of authorship css, I will refactor other css in this PR too and update it accordingly. Let me know.

File: frontend/src/index.pug [hunk 1]
```diff
@@ -8,6 +8,7 @@ html
     link(rel="stylesheet", href="https://cdn.jsdelivr.net/npm/highlight.js@9.14/styles/color-brewer.min.css")
     link(rel="stylesheet", href="https://cdn.jsdelivr.net/npm/vue-loading-spinner@1.0/dist/vue-loading-spinner.css")
     link(rel="stylesheet", href="static/css/style.css")
+    link(rel="stylesheet", href="static/css/authorship.css")
```
Comment (fzdy1914) [inline frontend/src/index.pug]: Let the name be `v_authorship.css`

**Comment by Tejas2805** [comment]
> My concern is that, if we make the splitting in one PR, there will be a lot of merge conflicts during the review process and after it is merged.

Good point. Maybe we can split it into three PRs: 
1. Authorship
2. Summary and Summary Charts
3. Ramp and zoom

**Comment by Tejas2805** [comment]
@fzdy1914 Any other changes required?

**Pull request merged**